### PR TITLE
Add the new endpoints to the restapi draft

### PR DESCRIPTION
**Pull request by SarahBA**

### CODE REVIEW

File: b2share/modules/main/restful.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    def get(self, user_id, **kwargs):
+        """
+        Get a user profile information
+        """
+        return None
+
+
+def setup_app(app, api):
+    api.add_resource(CommunityList, '/api/communities')
+    api.add_resource(CommunityRes, '/api/communities/create')
```
Comment (emanueldima) [inline b2share/modules/main/restful.py]: We've decided not to use verbs in the URLs, so this one must go. The same functionality can be achieved by a POST in CommunityList.

File: b2share/modules/main/restful.py [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    api.add_resource(Community, '/api/communities/<int:community_id>')
+    api.add_resource(schemaList, '/api/schemas')
+    api.add_resource(schemaRes, '/api/schemas/create')
```
Comment (emanueldima) [inline b2share/modules/main/restful.py]: No verbs, see above.

File: b2share/modules/main/restful.py [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+    api.add_resource(schema, '/api/schemas/<int:schema_id>')
+    api.add_resource(schemaDeprecate, '/api/schemas/<int:schema_id>/deprecate')
```
Comment (emanueldima) [inline b2share/modules/main/restful.py]: No verbs

File: b2share/modules/main/restful.py [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+        """
+        Modify a community
+        """
+        return None
+
+
+# ----------------------------------------------- SCHEMA
+
+
+class schemaList(Resource):
```
Comment (emanueldima) [inline b2share/modules/main/restful.py]: Let's agree to start all class names with Capital Letters: https://www.python.org/dev/peps/pep-0008/#class-names

**Comment by SarahBA** [comment]
Fixed the comments @emanueldima mentioned

**Pull request merged**